pkg/pillar/cmd/nim/nim.go

```diff
@@ -988,7 +988,7 @@ func handleGlobalConfigModify(ctxArg interface{}, key string,
 			} else {
 				log.Infof("Starting TestBetterTimer: %d",
 					gcpNetworkTestBetterInterval)
-				networkTestBetterInterval := time.Duration(ctx.deviceNetworkContext.NetworkTestBetterInterval) * time.Second
+				networkTestBetterInterval := time.Duration(gcpNetworkTestBetterInterval) * time.Second
```

Comment:
Do you think this old bug can explain why we see devices not getting back to using Index=0 for hours or days?

Reply:
No, this bug does not explain why better test timer does not pick up.
I'll look into that while working on the other PR that addresses delay in interface bringup.